dftimewolf/lib/processors/turbinia.py

```diff
@@ -5,11 +5,10 @@
 import os
 import tempfile
 
+from turbinia import TurbiniaException
```

Comment:
This is a style guide violation, but I guess you don't have a choice - otherwise you end up importing all of Turbinia http://google.github.io/styleguide/pyguide.html#22-imports. Please add a comment about why you're importing a class here, instead of a module